test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/individual/BusinessNameConfirmationViewSpec.scala

```diff
@@ -68,29 +74,50 @@ class BusinessNameConfirmationViewSpec extends ViewSpec {
 
   "Business Name Confirmation" must {
     "use the correct page template" when {
-      "there is no error on the page" in new TemplateViewTest(
-        page(),
-        title = BusinessNameConfirmationMessages.heading,
-        isAgent = false,
-        backLink = Some(testBackUrl),
-        hasSignOutLink = true
-      )
-      "there is an error on the page" in new TemplateViewTest(
-        page(hasFormError = true),
-        title = BusinessNameConfirmationMessages.heading,
-        isAgent = false,
-        backLink = Some(testBackUrl),
-        hasSignOutLink = true,
-        error = Some(testFormError.key -> BusinessNameConfirmationMessages.Form.emptyError)
-      )
+      "the page is for a personal name" when {
+        "there is no error on the page" in new TemplateViewTest(
+          page(),
+          title = BusinessNameConfirmationMessages.heading,
+          backLink = Some(testBackUrl),
+          hasSignOutLink = true
+        )
+        "there is an error on the page" in new TemplateViewTest(
+          page(hasFormError = true),
+          title = BusinessNameConfirmationMessages.heading,
+          backLink = Some(testBackUrl),
+          hasSignOutLink = true,
+          error = Some(testFormError.key -> BusinessNameConfirmationMessages.Form.emptyError)
+        )
+      }
+
+      "the page is for a secondary business" when {
+        "there is no error on the page" in new TemplateViewTest(
+          page(isBusinessName = true),
+          title = BusinessNameConfirmationMessages.headingSecond,
+          backLink = Some(testBackUrl),
+          hasSignOutLink = true
+        )
+        "there is an error on the page" in new TemplateViewTest(
+          page(hasFormError = true, isBusinessName = true),
+          title = BusinessNameConfirmationMessages.headingSecond,
+          backLink = Some(testBackUrl),
+          hasSignOutLink = true,
+          error = Some(testFormError.key -> BusinessNameConfirmationMessages.Form.emptyError)
+        )
+      }
     }
 
+
+
+
+
     "have a page heading with caption" in {
       val heading: Element = document().mainContent.selectHead(".hmrc-page-heading")
       heading.selectHead("h1.govuk-heading-l").text() mustBe BusinessNameConfirmationMessages.heading
       heading.selectHead("p.govuk-caption-l").text() mustBe BusinessNameConfirmationMessages.caption
     }
```

Comment:
You have a split of tests for the template + title, we probably want the same here for the heading:
```
"have the page heading with caption" when {
  "the page is for a personal name" in {...}
  "the page is for a secondary business" in {...}
}
```